Website/readme.md

```diff
@@ -1 +1,5 @@
-This website is used to help display the Student Council's duty list in a organised and neat way.
\ No newline at end of file
+This website is used to help display the Student Council's duty list in a organised and neat way.
+
+**Versions:**
+-  Bootstrap: **v5.0.2**
+-  Jquery: **v3.6.0**
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8389313)
```suggestion
- Jquery: **v3.6.0**
```